### PR TITLE
Update push, terminology and nav

### PR DESCRIPTION
**Pull request by finp**

### CODE REVIEW

File: modules/ROOT/pages/_partials/generic-binding.adoc [hunk 1]
```diff
@@ -0,0 +1,20 @@
+include::{partialsdir}/attributes.adoc[]
+
+After binding a {mobile-client} to the {service-name} service, your mobile app can use that service immediately. 
+
+: The association between a {mobile-client} and {service-name} service uses the xref:https://blog.openshift.com/asynchronous-bind-with-the-automation-broker/[OpenShift bind] feature.
+
+To associate a {mobile-client} with a mobile service:
+
+. Navigate to the *Overview* your OpenShift project.
```
Comment (jstaffor) [inline modules/ROOT/pages/_partials/generic-binding.adoc]: Grammar

File: modules/ROOT/pages/_partials/generic-binding.adoc [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+. Expand the {mobile-client} you want to associate by clicking the *>* icon.
```
Comment (jstaffor) [inline modules/ROOT/pages/_partials/generic-binding.adoc]: you want to associate to the service by clicking

File: modules/ROOT/pages/_partials/generic-binding.adoc [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
++
+NOTE: Your mobile app will be represented by two items in OpenShift, an item in the *{mobile-client}s* list and an item in the *Provisioned Services* list. Make sure to expand the item in the *{mobile-client}s* list.
+
+. Note the value for `clientId` in the *Config Info* field. You will need this for the next step.
```
Comment (jstaffor) [inline modules/ROOT/pages/_partials/generic-binding.adoc]: " You will " - this is required

File: modules/ROOT/pages/_partials/generic-binding.adoc [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+. Choose the {service-name} service from the *Integrations* list. If an option is not available, you can provision that service by choosing that option and associate the {mobile-client} later.
```
Comment (jstaffor) [inline modules/ROOT/pages/_partials/generic-binding.adoc]: "you can" - can we drop this?

File: modules/ROOT/pages/_partials/generic-binding.adoc [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+. Follow the *Create Binding* wizard to associate the {mobile-client} with the {service-name} service. You may be prompted for a *{mobile-client} ID*. Enter the value you noted from the previous step.
```
Comment (jstaffor) [inline modules/ROOT/pages/_partials/generic-binding.adoc]: "You may be prompted for a *{mobile-client} ID*. Enter the value you noted from the previous step." --> If prompted for id...

File: modules/ROOT/pages/metrics/manually-adding-services-to-metrics.adoc [hunk 6]
```diff
@@ -1,3 +1,5 @@
+include::{partialsdir}/attributes.adoc[]
+
 = Manually making a service visible to the Metrics Service
```
Comment (jstaffor) [inline modules/ROOT/pages/metrics/manually-adding-services-to-metrics.adoc]: Is the word "Manually" needed?

File: modules/ROOT/pages/metrics/manually-adding-services-to-metrics.adoc [hunk 7]
```diff
@@ -47,22 +49,22 @@ documentation on scrape_config] for more details.
 
 === Via the OpenShift Console
 
-1.  Select your project in the `My Projects` sidebar
+.  Select your project in the `My Projects` sidebar
```
Comment (jstaffor) [inline modules/ROOT/pages/metrics/manually-adding-services-to-metrics.adoc]: Should `My Projects` be /* My Projects /* ?

File: modules/ROOT/pages/metrics/manually-adding-services-to-metrics.adoc [hunk 8]
```diff
@@ -47,22 +49,22 @@ documentation on scrape_config] for more details.
...
 
 image::manually-adding-services-to-metrics/myprojects.png[My
 projects]
 
-2.  Navigate to Prometheus' Config Map via
+.  Navigate to Prometheus' Config Map via
```
Comment (jstaffor) [inline modules/ROOT/pages/metrics/manually-adding-services-to-metrics.adoc]: ' ?

File: modules/ROOT/pages/mobile_terminology.adoc [hunk 9]
```diff
@@ -2,25 +2,26 @@ include::{partialsdir}/attributes.adoc[]
...
-[[mobile-client]]{mobile-client}:: A {mobile-client} is the representation on OpenShift of the Mobile App that you want to implement.
-For more information, see xref:workflow:clients.adoc[Provisioning and Configuring {mobile-client}s].
 
-[[mobile-app]]Mobile App:: A Mobile App is your code for your Mobile Platform.
-For more information, see xref:workflow:apps.adoc[Provisioning and Configuring {mobile-client}s].
+[[mobile-service]]Mobile Service:: A mobile service accelerates your mobile app development by providing solutions for common mobile development use cases such as Push notifications. Mobile services are hosted on OpenShift and you use the OpenShift console to provision and associate services with your mobile apps.
+For more information, see xref:con_services.adoc[Mobile Services].
 
-Binding a Service to a Mobile App:: You can associate a Mobile Service with a Mobile App to allow your Mobile Apps to communicate with that service.
+[[mobile-client]]{mobile-client}:: A {mobile-client} is the representation the mobile app in an OpenShift project. This allows you connect and disconnect an app with any mobile service in the same OpenShift project.
```
Comment (jstaffor) [inline modules/ROOT/pages/mobile_terminology.adoc]: mobile app - caps (I think we defo need agreement on this as it's littered across the docs)?

an app - mobile app?

mobile service - Mobile Service?

File: modules/ROOT/pages/_partials/generic-binding.adoc (hunk 4 above)
Comment (finp) [reply modules/ROOT/pages/_partials/generic-binding.adoc]: leaving it for now

File: modules/ROOT/pages/metrics/manually-adding-services-to-metrics.adoc (hunk 6 above)
Comment (finp) [reply modules/ROOT/pages/metrics/manually-adding-services-to-metrics.adoc]: whole topic needs review

File: modules/ROOT/pages/metrics/manually-adding-services-to-metrics.adoc (hunk 7 above)
Comment (finp) [reply modules/ROOT/pages/metrics/manually-adding-services-to-metrics.adoc]: whole topic needs review

File: modules/ROOT/pages/metrics/manually-adding-services-to-metrics.adoc (hunk 8 above)
Comment (finp) [reply modules/ROOT/pages/metrics/manually-adding-services-to-metrics.adoc]: whole topic needs review

File: modules/ROOT/pages/mobile_terminology.adoc (hunk 9 above)
Comment (finp) [reply modules/ROOT/pages/mobile_terminology.adoc]: lower case whenever possible

**Pull request merged**